sdk/test-utils/recorder/tsconfig.json

```diff
@@ -2,6 +2,7 @@
   "extends": "../../../tsconfig.package",
   "compilerOptions": {
     "declarationDir": "./typings",
+    "target": "es5",
```

Comment:
What is the step that failed without this? 
Running the tests in playback in IE or even the live tests?

Reply:
yes running playback test in IE

Reply:
could be both, since the bundle contained `class BaseRecorder` I remember.